framework/doc/content/syntax/Executioner/FixedPointAlgorithms/index.md

```diff
@@ -0,0 +1,61 @@
+# Fixed point iteration algorithms
+
+MOOSE provides fixed point algorithms in all its executioners. This can be used to iterate a single
+application solve to converge a parameter, for example converge the mass flow rate of a fluid simulation with a target pressure drop.
+But it is more often used to tightly couple multiphysics simulations, where the `MultiApp` system is leveraged to couple
+two different problems, and iterating each application, transferring information between each solve, brings the coupling to convergence.
+
+Within one app coupling iteration, MultiApps executed on `TIMESTEP_BEGIN`, the main app and MultiApps executed on `TIMESTEP_END` are executed, in that order.
+The execution order of MultiApps within one group (`TIMESTEP_BEGIN` or `TIMESTEP_END`) is undefined.
+The relevant data transfers happen before and after each of the two groups of MultiApps runs.
+Because MultiApp allows wrapping another levels of MultiApps, the design enables multi-level app coupling iterations automatically.
+
+Regardless of the fixed point algorithm used, solution vectors can be relaxed to improve the stability of the convergence.
+When a MultiApp is a sub-app and the main application of its own sub-apps, MOOSE allows relaxation of the MultiApp solution
```

Comment:
This sentence is quit hard to understand, maybe rephrase a little bit?

Reply:
changed